geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -38,6 +39,114 @@ app.module.directive('appMymaps', app.mymapsDirective);
  * @constructor
  * @export
  */
-app.MymapsDirectiveController = function() {};
+app.MymapsDirectiveController = function() {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.defaultTitle_ = 'Map without title';
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.modifying = false;
```

Comment:
It would be good to add some description to know what this state variable tracks.